.github/workflows/e2e.yml

```diff
@@ -14,15 +14,15 @@ jobs:
     runs-on: ubuntu-latest-16-cores
     env:
       # the gh tag of system-test repo version to run 
-      SYSTEM_TEST_GIT_REF: v1.0.8
+      SYSTEM_TEST_GIT_REF: 3f54b84543126967620f8f00ac043c12e477ab6e
 
       # the soroban tools source code to compile and run from system test
       # refers to checked out source of current git hub ref context 
       SYSTEM_TEST_SOROBAN_TOOLS_REF: ${{ github.workspace }}/soroban-tools
 
       # core git ref should be latest commit for stable soroban functionality
       # the core bin can either be compiled in-line here as part of ci, 
-      SYSTEM_TEST_CORE_GIT_REF: https://github.com/stellar/stellar-core.git#871accefc0c4a703c1321b4f0e48cf6e03b41960
+      SYSTEM_TEST_CORE_GIT_REF: https://github.com/stellar/stellar-core.git#53ea43ace0c72a3177a1c387f7a896e82fb0f1af
```

Comment:
@paulbellamy , is this the preview 8 release version of core?  I thought it was `064a2787acb9e98c70567523785333581ee1ffa4`, seen on last posting from the release channel

I didn't notice this pr earlier, I had opened up a similar one for preview 8 release, version bumping sys test to rel tag and core to that different ref  - https://github.com/stellar/soroban-tools/pull/533, lmk, can close that one or this one?